ADR/0011-roles-and-permissions.md

```diff
@@ -0,0 +1,81 @@
+|               | IntegrationTestScenario | appstudio.redhat.com      | *                                       | integrationtestscenarios
+|               | Enterprise contract     | appstudio.redhat.com      | get, list, watch                        | enterprisecontractpolicies
+|               | Release Strategy        | appstudio.redhat.com      | *                                       | releases, releasestrategies, releaseplans
+|               | Release Admission Plan  | appstudio.redhat.com      | *                                       | releaseplanadmissions
+|               | *JVM Build Service*     | jvmbuildservice.io        | get, list, watch                        | jbsconfigs, artifactbuilds
+|               | *Service Access*        | appstudio.redhat.com      | get, list, watch                        | spiaccesstokenbindings, spiaccesschecks, spiaccesstokens, spiaccesstokendataupdates
+|               | *Configs*               |                           | get, list, watch                        | configmaps
+|               | *Secrets*               |                           |                                         | secrets
+|               | Add User                |
+|               | User group (with SSO)   |
+| Owner         | Workspace               | Access to namespaces that backs workspace                           |
+|               | Application & Component | appstudio.redhat.com      | * | applications, components, componentdetectionqueries
+|               | Environment             | appstudio.redhat.com      | *                                       | promotionruns, snapshotenvironmentbindings, snapshots, environments
+|               | *GitOps*                | managed-gitops.redhat.com | get, list, watch                        | gitopsdeployments
+|               | PipelineRun             | tekton.dev                | *                                       | pipelineruns
+|               | Pipeline Results        | results.tekton.dev        | get, list                               | results, records
+|               | IntegrationTestScenario | appstudio.redhat.com      | *                                       | integrationtestscenarios
+|               | Enterprise contract     | appstudio.redhat.com      | *                                       | enterprisecontractpolicies
+|               | Release Strategy        | appstudio.redhat.com      | *                                       | releases, releasestrategies, releaseplans
+|               | Release Admission Plan  | appstudio.redhat.com      | *                                       | releaseplanadmissions
+|               | *JVM Build Service*     | jvmbuildservice.io        | get, list, watch                        | jbsconfigs, artifactbuilds
+|               | *Service Access*        | appstudio.redhat.com      | *                                       | spiaccesstokenbindings, spiaccesschecks, spiaccesstokens, spiaccesstokendataupdates
```

Comment:
spiaccesstokendataupdates is not used by User (SA only), can be removed
spifilecontentrequest should be added